### ISSUE TITLE
Course FAQ

### ISSUE THREAD

**Issue by FG-CTRI** [LMLhub/ee_lecture_notes#18]
See below, first pass at the course FAQs that come to mind. I am also keeping a "Here Be Dragons" FAQ that may be of interest. 

Starting Sunday, I will attend our annual IWI/RMA conference and will go silent until Thursday. I will be able to see e-mail and text only. I hope this work helps and look forward to seeing the next version of the Course Notes.

[2019.CTRI.EE.GitHub.Course Related FAQs.docx](https://github.com/LMLhub/ee_lecture_notes/files/3932354/2019.CTRI.EE.GitHub.Course.Related.FAQs.docx)

**Comment by unbalancedparentheses** [comment]
Excellent work, but please let's use markdown documents to discuss instead of office documents/ Many of us don't use office and it is not easy to track changes on those documents. I will create pull request with the markdown of the FAQ so that we can discuss the questions and answers.

**Comment by unbalancedparentheses** [comment]
I have opend #19, let's discuss it on the PR

**Comment by FG-CTRI** [comment]
Very helpful. Thank you for the guidance. 

I wonder if we will end-up writing several flavors of FAQs: 
FAQ for the Press and the Public, 
FAQ for Students of EE, 
FAQ for what's next with EE, etc.

**Comment by unbalancedparentheses** [comment]
I agree, that is a good idea. On #19 let's start working on the FAQ for press and the public and let's do the other ones afterwards

**Comment by FG-CTRI** [comment]
Here is may take on the start of a public/press FAQ. See below. Note: Use conceptual build-ups that chain ideas from one Q/A to another with What/Why/How/When/Who. I will be off line till next Thursday. Looking forward to catching up then.

Public FAQ

What is Ergodic Economics (EE)?
EE is a mathematical model of real-world growth patterns that changes our understanding of economics and finance in fundamental and useful ways.

Why does EE change our understanding of economics and finance?
EE changes our understanding of economics and finance because it fixes a 350-years old, foundational, error in calculation.

How does EE fixe this foundational error in calculation at the foundation of finance and economics?
EE applies new mathematics that were not available 350 years ago.

When did EE start its development?
EE was started by Ole Peters and Alex Adamou about 10 years ago...

**Comment by FG-CTRI** [comment]
Downloaded Live Tex.

Starting to learn it.

Back next Thursday.
